tests/mocks.py

```diff
@@ -235,8 +236,12 @@ def revision(
             "ccs": [],
             "hashes": [],
             "bugzilla.bug-id": bug_id,
+            "uplift.request": uplift,
             "repositoryPHID": repo["phid"] if repo is not None else None,
-            "fields": {"repositoryPHID": repo["phid"] if repo is not None else None},
+            "fields": {
+                "repositoryPHID": repo["phid"] if repo is not None else None,
+                "uplift.request": uplift,
+            },
```

Comment:
I added to the main dict in the same way as `bugzilla.bug-id`, but the object doesn't return as expected. Not sure if we test `bugzilla.bug-id` but perhaps it's supposed to go in the `fields` dict?